tools/xtce_generation/xtce_generator_glows.py

```diff
@@ -1,10 +1,21 @@
+"""
+Module for generating XTCE files for telemetry packets for glows.
+
+This module provides functionality to generate XTCE files for
+telemetry packets for glows.
+It includes a `TelemetryGenerator` class for creating XTCE
+files based on packet definitions stored in
+an Excel file.
```

Comment:
```suggestion
Module for generating XTCE files for telemetry packets for glows.

This module provides functionality to generate XTCE files for telemetry packets
for glows. It includes a `TelemetryGenerator` class for creating XTCE files
based on packet definitions stored in an Excel file.
```

For consistency across the repo, we should switch this (and in the other `xtce_generator_*` files) to use up all of the 80-character space.